### ISSUE TITLE
Where is the Scaladoc?

### ISSUE THREAD

**Issue by mslinn** [pureconfig/pureconfig#288]
Google showed me [this location](https://www.javadoc.io/doc/com.github.pureconfig/pureconfig-akka_2.12/0.7.2), but the Scaladoc is empty. If Scaladoc is not currently published, why not put Scaladoc in the `gh-pages` branch of this repo? I have found the following to work well:

Create the `gh-pages` branch with [this script](https://gist.github.com/mslinn/d7529a687df9479d93c4ba9b9d4b2e4b) - you will have to edit it for your information.

`project/plugins.sbt`
```
addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.2.0")
addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.0")
```
`publish.sbt`
```
val vcs = "github.com"

// sbt-site settings
enablePlugins(SiteScaladocPlugin)
siteSourceDirectory := target.value / "api"
publishSite

// sbt-ghpages settings
enablePlugins(GhpagesPlugin)
git.remoteRepo := s"git@$vcs:mslinn/${ name.value }.git"
```

To run:
```
$ sbt "; doc; ghpagesPushsite"

**Comment by leifwickland** [comment]
@ruippeixotog pointed out that this is due to a bug in scaldoc: https://github.com/scala/bug/issues/8124

**Comment by mslinn** [comment]
Odd I've never encountered that bug. I'm sure I've provided the circumstances. You could fix it like this:

```
/** Scaladoc goes here */
package object blah {
  private val justForScaladocToWork = true
}
```
... and then add the link to the Scaladoc to `README.md`

**Comment by ruippeixotog** [comment]
Closing as this is not a PureConfig bug and will get fixed automatically once scala/bug#8124 is fixed.
